kong/plugins/request-transformer/schema/migrations.lua

```diff
@@ -0,0 +1,70 @@
+local next = next
+
+local Migrations = {
+  {
+    name = "2016-03-10-161400_request-transformer-config",
+    up = function(options, dao_factory)
+
+      local plugins, err = dao_factory.plugins:find_by_keys {name = "request-transformer"}
+      if err then
+        return err
+      end
+
+      for _, plugin in ipairs(plugins) do
+        for _, action in ipairs {"remove", "add", "append", "replace"} do
+
+          if plugin.config[action] == nil then
+            plugin.config[action] = {}
```

Comment:
`plugin.config[action] = plugin.config[action] or {}` will be more concise and more consistent with l21